scripts/quests/otherAreas/Elderly_Pursuits.lua

```diff
@@ -0,0 +1,183 @@
+-----------------------------------
+-- Elderly Pursuits
+-----------------------------------
+-- Log ID: 4, Quest ID: 75
+-- Despachiaire : !pos 108 -40 -83 26
+-- Rouva        : !pos -16 2 11 230
+-- ???          : !pos -412 0 -362 2
+-----------------------------------
+-- !addquest 4 75
+-----------------------------------
+require('scripts/globals/interaction/quest')
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+-----------------------------------
+local carpentersLandingID = zones[xi.zone.CARPENTERS_LANDING]
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.ELDERLY_PURSUITS)
+
+local function AllParaDead()
+    for i = 0, 4 do
+        local mob = GetMobByID(carpentersLandingID.mob.PARA_OFFSET + i)
+        if not mob:isDead() then
+            return false
+        end
+    end
+
+    return true
+end
+
+quest.reward =
```

Comment:
Minor, but typically reward in all other scripts precedes any other functions or definitions (right after the quest is declared)